pkg/api/validation/schema.go

```diff
@@ -294,7 +294,7 @@ func (s *SwaggerSchema) isGenericArray(p swagger.ModelProperty) bool {
 }
 
 // This matches type name in the swagger spec, such as "v1.Binding".
-var versionRegexp = regexp.MustCompile(`^v.+\..*`)
+var versionRegexp = regexp.MustCompile(`^(v.+|unversioned)\..*`)
```

Comment:
@nikhiljindal @pwittrock I think you guys know this file-- is this a good way to fix the problem?